### PR TITLE
Image upload

### PR DESCRIPTION
**Pull request by angelk**
Fix for https://github.com/trackhub/website/issues/71

- [x] ToS update - not update needed
- [x] Thumbnail generation
- [x] hide image div if there are no images

### CODE REVIEW

**Comment by StefanMavrodiev** [comment]
@angelk 
Please merge with master and resolve conflicts.

**Comment by angelk** [comment]
merged

File: assets/js/app.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ require('bootstrap');
 // var $ = require('jquery');
 
 window.jQuery = require('jquery');
+window.bootstrap = require('bootstrap');
```
Comment (StefanMavrodiev) [inline assets/js/app.js]: Where is this used?

File: templates/base.html.twig [hunk 2]
```diff
@@ -58,6 +58,15 @@
...
 
+        <script>
+            jQuery(document).on('click', '[data-toggle="lightbox"]', function(event) {
+                event.preventDefault();
+                jQuery(this).ekkoLightbox({
+                    alwaysShowClose: true
+                });
+            });
+        </script>
+
```
Comment (StefanMavrodiev) [inline templates/base.html.twig]: Shouldn't this be in view..html.twig? Otherwise can we move it into foot.html.twig?

File: assets/css/app.css [hunk 3]
```diff
@@ -1,3 +1,30 @@
 body {
     background-color: lightgray;
 }
+
```
Comment (StefanMavrodiev) [inline assets/css/app.css]: Can we move this to separate file?

File: src/Entity/Track/Image.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Track", inversedBy="optimizedPoints")
+     */
+    private $track;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Track\Version", inversedBy="points")
+     */
+    private $version;
```
Comment (StefanMavrodiev) [inline src/Entity/Track/Image.php]: Where is the inversed side annotation?

File: src/Entity/Track/Image.php [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        $this->track = $track;
+        $track->addImage($this);
+    }
+
+    public function getId(): ?string
+    {
+        return $this->id;
+    }
+
+    public function setVersion(Version $version = null)
```
Comment (StefanMavrodiev) [inline src/Entity/Track/Image.php]: Unused function?

File: assets/js/app.js (hunk 1 above)
Comment (angelk) [reply assets/js/app.js]: 1) I will fix it
2) Please provide screenshot

File: templates/base.html.twig (hunk 2 above)
Comment (angelk) [reply templates/base.html.twig]: moved

File: src/Entity/Track/Image.php (hunk 5 above)
Comment (angelk) [reply src/Entity/Track/Image.php]: removed

File: assets/css/app.css (hunk 3 above)
Comment (angelk) [reply assets/css/app.css]: moved

File: src/Entity/Track/Image.php (hunk 4 above)
Comment (angelk) [reply src/Entity/Track/Image.php]: removed

File: assets/js/app.js (hunk 1 above)
Comment (angelk) [reply assets/js/app.js]: Bootstrap removed

**Pull request merged**